etc/inc/filter_log.inc

```diff
@@ -45,33 +45,36 @@ function conv_log_filter($logfile, $nentries, $tail = 50, $filtertext = "", $fil
 	global $config, $g;
 
 	/* Make sure this is a number before using it in a system call */
-	if (!(is_numeric($tail)))
+	if (!(is_numeric($tail))) {
 		return;
+	}
 
-	if ($filtertext)
+	if ($filtertext) {
 		$tail = 5000;
+	}
 
 	/* Always do a reverse tail, to be sure we're grabbing the 'end' of the log. */
 	$logarr = "";
 
-	if(isset($config['system']['usefifolog']))
+	if (isset($config['system']['usefifolog'])) {
 		exec("/usr/sbin/fifolog_reader " . escapeshellarg($logfile) . " | /usr/bin/grep 'filterlog:' | /usr/bin/tail -r -n {$tail}", $logarr);
-	else
+	} else {
 		exec("/usr/local/sbin/clog " . escapeshellarg($logfile) . " | grep -v \"CLOG\" | grep -v \"\033\" | /usr/bin/grep 'filterlog:' | /usr/bin/tail -r -n {$tail}", $logarr);
+	}
 
 	$filterlog = array();
 	$counter = 0;
 
 	$filterinterface = strtoupper($filterinterface);
 	foreach ($logarr as $logent) {
-		if($counter >= $nentries)
+		if ($counter >= $nentries) {
 			break;
+		}
 
 		$flent = parse_filter_line($logent);
-		if (!$filterinterface || ($filterinterface == $flent['interface']))
-		{
-			if ( ( ($flent != "") && (!is_array($filtertext)) && (match_filter_line ($flent, $filtertext))) || 
-			     ( ($flent != "") && ( is_array($filtertext)) && (match_filter_field($flent, $filtertext)) ) ) {
+		if (!$filterinterface || ($filterinterface == $flent['interface'])) {
+			if ((($flent != "") && (!is_array($filtertext)) && (match_filter_line ($flent, $filtertext))) ||
+				(($flent != "") && ( is_array($filtertext)) && (match_filter_field($flent, $filtertext)))) {
```

Comment:
Here is another case as I commented out on filter.inc. I'll re-review #1515, #1519 and #1520 and look for similar cases